src/Context/PlaylistContext.jsx

```diff
@@ -0,0 +1,100 @@
+import { createContext } from "react";
+import { useContext, useState,useEffect } from "react";
+import axios from "axios";
+import { toast } from "react-toastify";
+const notify = (message) => {
+  toast(message)   
+}
```

Comment:
instead of writing this function in all pages, you can create a separate file for it and call it here